AppCenter/AppCenter/Internals/Ingestion/MSIngestionProtocol.h

```diff
@@ -26,6 +26,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, readonly, getter=isReadyToSend) BOOL readyToSend;
 
+/*
+ * TODO We use sendAsync for both GET and POST but it doesn't really makes sense for a GET to send data.
```

Comment:
Is there a task for this? If not, let's now, next, or never it :) (AKA make the change here, create/reference a bug on the board, or remove the TODO)

Personally, I think the current name is fine because whether it is a `GET`, `POST`, or anything else, it's sending a **_request_**

Reply:
The problem is to define `GET`, `POST` or else we need to overwrite the `HTTPIngestion` "createRequest". Which means an `HTTPIngestion` object is tied to an HTTP method (btw we are abusing the term "ingestion"). I would like more flexibility and give this control to the caller. Plus ,`sendAsync:nil...` it resonates like sending emptiness to me :sweat_smile:.

I agree with the process (now, next, or never).

Reply:
Let's remove the TODO.